src/vs/platform/workspaces/electron-main/workspacesHistoryMainService.ts

```diff
@@ -162,31 +161,10 @@ export class WorkspacesHistoryMainService extends Disposable implements IWorkspa
 		this._onDidChangeRecentlyOpened.fire();
 	}
 
-	async getRecentlyOpened(include?: ICodeWindow): Promise<IRecentlyOpened> {
+	async getRecentlyOpened(): Promise<IRecentlyOpened> {
 		const workspaces: Array<IRecentFolder | IRecentWorkspace> = [];
 		const files: IRecentFile[] = [];
 
-		// Add current workspace to beginning if set
-		if (include) {
```

Comment:
I am just wondering are you removing this logic of adding current workspace to beginning intentionally or by mistake.

Reply:
Ok. Thanks for explanation.